frontend/src/EditUserProfile.tsx

```diff
@@ -15,8 +15,9 @@ import axios, { AxiosError } from "axios";
 import { useState } from "react";
 import { z } from "zod";
 import { userWithTimetablesType } from "../../lib/src/index";
+import { authenticatedRoute } from "./AuthenticatedRoute";
```

Comment:
```suggestion
import authenticatedRoute from "./AuthenticatedRoute";
```
Based on the above suggestion to fix the import.